src/openms_ml/source/ML/main.cpp

```diff
@@ -0,0 +1,31 @@
+#include <torch/torch.h>
+#include <torch/script.h>
+#include <vector>
+#include <OpenMS/ML/AlphapeptdeepWrapper.h>
+#include <OpenMS/ML/AlphaDatahandling.h>
+using namespace OpenMS;
+
+int main()
```

Comment:
I need to learn how I can make the test cases in CMake OpenMS environment? as I do mostly from main.
any suggestions please. @jpfeuffer @timosachsenberg